G1ANT.Addon.UI/Enums/EventTypes.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace G1ANT.Addon.UI.Enums
+{
+    public enum EventTypes
+    {
+        Unknown,
+        KeyUp,
+        KeyDown,
+        KeyPress,
+        MouseLeftClick = 9000,
+        MouseRightClick = 9001,
+        MouseDoubleClick = 9002
+    }
```

Comment:
Why values like this (9000-9002)? Is it to make it consistent with other enums from external libraries? Or just made up values?